File: dotnet/Sers/Sers.Core/Sers.Core.Station/Module/Api/ApiDesc/SsModelEntity.cs
using System.Collections.Generic;
using Newtonsoft.Json;

namespace Sers.Core.Module.Api.ApiDesc
{
    [JsonObject(MemberSerialization.OptIn)]
    public class SsModelEntity
    {


        /// <summary>
        /// 数据类型。可以唯一定位到一个模型
        /// </summary>
        [JsonProperty]
        public string type;

        /// <summary>
        ///  数据模式。只可为 value、object、array
        /// </summary>
        [JsonProperty]
        public string mode;

 


        [JsonProperty]
        public List<SsModelProperty> propertys;
    }
}

File: java/Sers.ServiceStation/src/Sers/Core/Module/Api/ApiDesc/SsModelEntity.java
package Sers.Core.Module.Api.ApiDesc;

public class SsModelEntity {

    /**
         * 数据类型。可以唯一定位到一个模型
     */
    public String type;

    /**
         * 数据模式。只可为 value、object、array   
     */
    public String mode;

 
    public SsModelProperty[] propertys;
}
